src/sentry/static/sentry/app/utils/integrationUtil.tsx

```diff
@@ -32,8 +32,17 @@ export const clearAnalyticsSession = () => {
 export const getAnalyticsSessionId = () =>
   window.sessionStorage.getItem(INTEGRATIONS_ANALYTICS_SESSION_KEY);
 
-export const getCategorySelectActive = () =>
-  localStorage.getItem(SHOW_INTEGRATION_DIRECTORY_CATEGORY_SELECT) === '1';
+export const getCategorySelectActive = (organization?: Organization) => {
+  const variant = organization?.experiments?.IntegrationDirectoryCategoryExperiment;
+  switch (localStorage.getItem(SHOW_INTEGRATION_DIRECTORY_CATEGORY_SELECT)) {
+    case '1':
+      return true;
+    case '0':
+      return false;
+    default:
+      return variant && variant === '1';
+  }
```

Comment:
Why a switch case instead of just an if else?

```jsx
const enabledViaLocalStore = localStorage.getItem(SHOW_INTEGRATION_DIRECTORY_CATEGORY_SELECT)
if (enabledViaLocalStore !== undefined) {
  return enabledViaLocalStore === '1';
} 

return organization?.experiments?.IntegrationDirectoryCategoryExperiment === '1';
```